aws/table_aws_s3_object.go

```diff
@@ -0,0 +1,668 @@
+				Transform:   transform.FromField("WebsiteRedirectLocation"),
+				Hydrate:     getS3Object,
+			},
+			{
+				Name:        "acl",
+				Description: "ACLs define which AWS accounts or groups are granted access along with the type of access.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromValue(),
+				Hydrate:     getS3ObjectACL,
+			},
+			{
+				Name:        "checksum",
+				Description: "The checksum or digest of the object.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Checksum"),
+				Hydrate:     getS3ObjectAttributes,
+			},
+			{
+				Name:        "metadata",
+				Description: "A map of metadata to store with the object in S3.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Metadata"),
+				Hydrate:     getS3Object,
+			},
+			{
+				Name:        "object_parts",
+				Description: "A collection of parts associated with a multipart upload.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("ObjectParts"),
+				Hydrate:     getS3ObjectAttributes,
+			},
+			{
+				Name:        "owner",
+				Description: "The owner of the object.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "tags_src",
+				Description: "A list of tags assigned to the object.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("TagSet"),
+				Hydrate:     getS3ObjectTagging,
+			},
+
+			// Steampipe standard columns
+			{
+				Name:        "tags",
+				Description: resourceInterfaceDescription("tags"),
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("TagSet").Transform(handleS3TagsToTurbotTags),
+				Hydrate:     getS3ObjectTagging,
+			},
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Key"),
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getObjectARN,
+				Transform:   transform.FromValue().Transform(transform.EnsureStringArray),
+			},
+			{
+				Name:        "region",
+				Description: "The AWS Region in which the object is located.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getBucketLocationForObjects,
+				Transform:   transform.FromValue(),
+			},
+		}),
+	}
+}
+
+func listS3Objects(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	// Bucket location will be nil if getBucketLocationForObjects returned an error but
+	// was ignored through ignore_error_codes config arg
+	location, err := getBucketLocationForObjects(ctx, d, h)
+	if err != nil {
+		return nil, err
+	} else if location == "" {
+		return nil, nil
+	}
+
+	svc, err := S3Client(ctx, d, fmt.Sprint(location))
+	if err != nil {
+		plugin.Logger(ctx).Error("listS3Objects", "get_client_error", err)
```

Comment:
```suggestion
		plugin.Logger(ctx).Error("aws_s3_object.listS3Objects", "get_client_error", err)
```
@bigdatasourav could you please append the table name to all the log statements?

Reply:
updated